tests/devices/test_default_mixed.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane.plugin.DefaultMixed` device.
```

Comment:
```suggestion
Unit tests for the :mod:`pennylane.devices.DefaultMixed` device.
```
The `plugin` is a remnant from before the folder was renamed.